ext/mysqli/tests/functions/mysqli_affected_rows.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+function test: mysqli_affected_rows
+--EXTENSIONS--
+mysqli
+--SKIPIF--
+<?php
+require_once(dirname(__DIR__) . "/test_setup/test_helpers.inc");
```

Comment:
While you are changing these tests I would really appreciate if you could remove the useless and confusing parentheses.